web/src/use/use-user.ts

```diff
@@ -1,7 +1,7 @@
-import { type Ref, reactive, unref, watch, toRefs } from "vue"
+import { type Ref, reactive, unref, watch, toRefs, computed } from "vue"
 import { isNil } from "lodash"
 
-import usersApi, { type User, type UserUpdate } from "@/api/users-api"
+import usersApi, { RoleTypes, type User, type UserUpdate } from "@/api/users-api"
 
 export { type User, type UserUpdate }
```

Comment:
I would also export `RoleTypes` here since it sometimes is more convenient to import it from the use file than the api. Note that `RoleTypes` isn't a "type` export, its just a regular one.